packages/strapi-parts/src/EmptyStateLayout/EmptyStateLayout.stories.mdx

```diff
@@ -20,14 +20,26 @@ EmptyStates are a pattern used to indicate a page or a table is empty.
 - Use the most appropriate illustration for the usecase. If related to permissions, choose the eye one.
 - Use the button to help redirecting the user to a more relevant place.
 
+[View source](https://github.com/strapi/parts/tree/develop/packages/strapi-parts/src/EmptyStateLayout)
+
+## Imports
+
+```js
+import { EmptyStateLayout } from '@strapi/parts/EmptyStateLayout';
+import { Button } from '@strapi/parts/Button';
+import AddIcon from '@strapi/icons/AddIcon';
+import { Illo } from '@strapi/parts/Illo';
```

Comment:
I don't think it's the right import it should be an icon from @strapi/icons

Reply:
I'm not sure what to do about this feedback.

Reply:
This Illo does not exist in the strapi/icons package. I think it's the EmptyStateDocument.